### PR TITLE
Add more details of sig advocacy and outreach

### PR DESCRIPTION
**Pull request by LinuxSuRen**
/cc @kubesphere/sig-advocacy-and-outreach

### CODE REVIEW

File: sig-advocacy-and-outreach/summer-ospp/README.md [hunk 1]
```diff
@@ -11,11 +11,22 @@ KubeSphere in [Summer 2021 of Open Source Promotion Plan](https://github.com/sum
...
+Please check out the following pages if you're a student who is interested in this program.
+
 * [Students application process](students-application-process.md)
 * [Open Source Promotion Plan Portal](https://summer.iscas.ac.cn/)
 
+## Administrators
+
+The following people are the administrators of this program in 2021. Please contact them if you have any questions.
+
+* [shenhonglei](https://github.com/shenhonglei)
```
Comment (LinuxSuRen) [inline sig-advocacy-and-outreach/summer-ospp/README.md:23]: hi @shenhonglei , please let me know what's your preferred name on this page.

Comment (shenhonglei) [reply sig-advocacy-and-outreach/summer-ospp/README.md:23]: shenhonglei

**Review by shenhonglei — APPROVED** [review]
OK

File: sig-advocacy-and-outreach/README.md [hunk 2]
```diff
@@ -17,8 +17,8 @@ and attract new people to join KubeSphere community.
 ## Topics
 
 * Attracting new individual and company contributors
-* Coordinating community outreach events: meetups, Hacktoberfest, [GSoC](./gsoc), etc.
-* Community Evangelism (KubeSphere Ambassadors, etc.)
+* Coordinating community outreach events: meetups, Hacktoberfest, [GSoC](./gsoc), [Summer of Open Source Promotion Plan](./summer-ospp), etc.
+* Community Evangelism ([KubeSphere Ambassadors](https://kubesphere.com.cn/en/contribution/request/), etc.)
```
Comment (FeynmanZhou) [inline sig-advocacy-and-outreach/README.md:21]: can we use `https://kubesphere.io/contribution/request/` here?

File: sig-advocacy-and-outreach/gsoc/README.md [hunk 3]
```diff
@@ -2,6 +2,10 @@ KubeSphere in [Google Summber of Code (aka GSoC)](https://summerofcode.withgoogl
 
 ## Proposals
 
+### Archived ideas
+
+Below ideas are not accepted by Google Summber of code. Please contact us if you're interested in them. For example, you can explore them through [Summer of Open Source Promotion Plan](../summer-ospp).
```
Comment (FeynmanZhou) [inline sig-advocacy-and-outreach/gsoc/README.md:7]: The ideas below

**Review by FeynmanZhou — APPROVED** [review]

**Pull request merged**